src/tasks/views.py

```diff
@@ -65,11 +65,11 @@ def get_queryset(self):
         return base_queryset.all()
 
 
-class MultipleFieldLookupMixin(object):
+class MultipleFieldLookupMixin:
     """Get multiple field filtering based on a `lookup_fields` attribute."""
 
     def get_queryset(self):
-        base_queryset = super(MultipleFieldLookupMixin, self).get_queryset()
+        base_queryset = super().get_queryset()
```

Comment:
Is this class missing a parent class?

Reply:
This was a change introduced in python 3.